### DISCUSSION TITLE
sliderfollowcircle appears almost instantly and doesn't pulse on sliderticks in lazer as opposed to stable.

### DISCUSSION THREAD

**Discussion by r0unded** [ppy/osu#18190] [Ideas] [answered]
In stable, sliderfollowcircle has a visible start and end animation of it growing/becoming smaller, whereas in lazer it almost appears in it's final form and disappears instantly. It also doesn't pulse on sliderticks. Both of those things noticably affect gameplay, but especially the start/end animation. Video comparisons:

https://user-images.githubusercontent.com/105223802/167435394-2437cea2-222c-4862-8c34-8570b4dfc104.mp4

https://user-images.githubusercontent.com/105223802/167435402-03cb77ae-8c14-4fdd-95be-b7f319da6c74.mp4

**Comment by peppy** [comment, marked as answer]
Please search first, this has been reported many times.
